### ISSUE TITLE
Solution to "Connect failed" of three-dimensional camera.

### ISSUE THREAD

**Issue by chenyangcheung** [chenyangcheung/QinchuanCattleBMS#53]
* Recheck the DHCP 
* Delete "New application" of O3D303
* Restart camera
* Restart computer

**Comment by Fibird** [comment]
I think you can write the solution in detail. And it's better to use figures.

**Comment by chenyangcheung** [comment]
Yeah, later, After I truly figure out how's that happen and how can I fix it. Last time, it's just my luck, I guess.
